tox.ini

```diff
@@ -65,7 +67,7 @@ commands =
 	 # fragment explaining the change in this branch.  Create one at
 	 # `newsfragments/<ticket>.<change type>` with some text for the news
 	 # file.  See pyproject.toml for legal <change type> values.
-	 python -m towncrier.check
+	 - python -m towncrier.check
```

Comment:
I don't understand this change.  What does it mean to have a `-` here?

Reply:
The dash makes the test pass even if the towncrier check fails. [Here](https://tox.readthedocs.io/en/latest/example/basic.html#ignoring-a-command-exit-code) is the tox documentation for it.

Reply:
I am going to undo this since tickets are required and thus towncrier should not fail.